front/app/utils/moduleUtils.tsx

```diff
@@ -485,32 +490,64 @@ export const RouteTypes = {
   ADMIN: 'admin',
 };
 
+const LoadingComponent = ({ type, children }) => {
+  if (type === RouteTypes.CITIZEN) {
+    return (
+      <React.Suspense fallback={LoadableLoadingCitizen}>
+        {children}
+      </React.Suspense>
+    );
+  }
+  return (
+    <React.Suspense fallback={LoadableLoadingAdmin}>{children}</React.Suspense>
+  );
+};
```

Comment:
Going to put this one on a separate branch so we can pull in from that branch and we use the same version.